client/src/components/AppContainer.js

```diff
@@ -13,6 +13,7 @@ import Onboarding from '../screens/Onboarding'
 import Login from '../screens/Login'
 import Loading from '../components/Loading'
 import Header from '../components/Header'
+import { navigateGroupProductById } from '../features/product'
```

Comment:
Redux shall not be used only for navigation purposes. Use react-navigation instead.

Reply:
I have already deleted navigateGroupProductById from develop branch

Reply:
navigation is fixed.
the answer is not reflected on the product page yet.